nilearn/image/resampling.py

```diff
@@ -393,7 +395,7 @@ def resample_img(img, target_affine=None, target_shape=None,
     # We now know that some resampling must be done.
     # The value of "copy" is of no importance: output is always a separate
     # array.
-    data = img.get_data()
+    data = _safe_get_data(img)
```

Comment:
So you are saying that we should leave the choice to load data from disk or not to the user? Do we need to harmonize this behavior?